docs/DeveloperGuide.md

```diff
@@ -158,21 +158,51 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Lesson Time Parameter
+This parameter allows users to keep track of a student's lesson timings. Multiple lesson times can be added for a single student.
+
+#### Design Considerations
+**Treatment of clashing timings**
+
+- **Current Implementation (Alternative 1):**
+    - **Description**: Clashing lesson timings across and within students are not rejected.
+    - **Pros**: Promotes greater user flexibility, as there could be different schedules for alternate weeks,
+      and all the timings should be stored in EduManage.
+    - **Cons**: Wrong lesson timings would not be flagged out to the user.
+
+- **Alternative 2:**
+    - **Description**: Reject clashing lesson timings across and within students.
+    - **Pros**: Flags out wrong lesson timings input by the user.
```

Comment:
wrong -> incorrect or conflicting
Ensures incorrect or conflicting timings are flagged, helping the user avoid scheduling errors.

Wrong feels a bit vague